UnityAds/build.gradle.kts

```diff
@@ -4,9 +4,9 @@ plugins {
 }
 
 private val versionMajor = 4
-private val versionMinor = 0
-private val versionPatch = 1
-private val versionAdapterPatch = 2
+private val versionMinor = 1
+private val versionPatch = 0
+private val versionAdapterPatch = 3
```

Comment:
Should the adapter patch go back down to 0? (I think so, so the adapter version is 4.1.0.0)